### ISSUE TITLE
Error when profile does not exist

### ISSUE THREAD

**Issue by bjornfreespee** [damoonrashidi/ladder#7]
KOP throws an error when doing

```
kop profile <profile that does not exist>
```

```
(node:47183) UnhandledPromiseRejectionWarning: RangeError: Invalid array length
    at Object.exports.plot (/Users/bjornax/.config/yarn/global/node_modules/asciichart/asciichart.js:29:22)
    at Command.program.command.description.action (/Users/bjornax/.config/yarn/global/node_modules/kingofpong/src/index.js:85:28)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
(node:47183) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:47183) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by damoonrashidi** [comment]
Thanks for letting us know @bjornfreespee! we'll have a look at this ASAP!

**Comment by arvidboivie** [comment]
This should be fixed with #14 👍
